common-healing.lic

```diff
@@ -113,22 +113,34 @@ module DRCH
     end
 
     parasites_regex = Regexp.union($DRCH_PARASITES_REGEX_LIST)
-    poisons_regex = Regexp.union([/^.* (?:has|have) a .* poisoned (?<part>[\w\s]*)/,
-                                  /^.* (?:has|have) a .* (?<part>internal) poison/,
-                                  /Cyanide poison/])
+
+    poisons_regex = Regexp.union([
+      /^[\w]+ (?:has|have) a .* poison/,
+      /having trouble breathing/,
+      /Cyanide poison/
+    ])
+
+    diseases_regex = Regexp.union([
+      /^[\w]+ wounds are infected/,
+      /^[\w]+ (?:has|have) a dormant infection/,
+      /^[\w]+ (?:body|skin) is covered in open oozing sores/
+    ])
 
     perceived_wounds = Hash.new { |h, k| h[k] = [] }
     perceived_parasites = Hash.new { |h, k| h[k] = [] }
     perceived_poison = false
     perceived_disease = false
     wound_body_part = nil
+
     health_lines
-      .take_while { |line| line != stop_line }
+      .take_while { |line| !line.casecmp?(stop_line) }
```

Comment:
@KatoakDR - I don't know that this actually solves the problem - because we're using `include?` above to make sure the stop line is exactly in the array.

Reply:
The only place I call 'perceive_health_other' is from a script that gets the names perfectly because it uses the room info. If we want to make it more resilient, should we be regexing the name out of the successful touch on 96?

Reply:
When I was testing I was calling the utility method from game command as `,e DRCH.perceive_health_other('someone')` and me not capitalizing the name was the issue. With above casecmp it worked for me.

Reply:
Though you’re right about the `if !health_lines.include?(stop_line)` should have blocked me. Or maybe I made the code tweak thinking that was the fix and continued to test with capitalized names. Now I can’t remember.

Reply:
> I don't know that this actually solves the problem

Good catch, I've pushed https://github.com/wstampley/dr-scripts/pull/4/commits/708772299ebcb7a47c907df9365a5489aa1737fb to apply a real fix, thanks